app/templates/partials/forms/input.html

```diff
@@ -1,14 +1,14 @@
 {% set attr = {
-  "class": "input input--" ~ input.type ~ " " ~ input.classes ~ " " ~ exclusive_class,
-  "type": type,
+  "class": "input input--" ~ input.type ~ " " ~ input.classes ~ " " ~ exclusive_class ~ " " ~ width_class,
+  "type": type if type is not none else input.type,
```

Comment:
Probably want: `"type": type or input.type`
I don't get why we have 2 variables that could hold the 'type', we should only have one?.

Line 2 references `input.type` but this can sometimes be invalid?

Reply:
I've seen a few places where type was used, but I'll have a look through the templates and change these instances to use an input dict like Andrew mentioned and remove just type all together